cli/release.go

```diff
@@ -0,0 +1,85 @@
+package cli
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+
+	"github.com/cloudflare/cfssl_trust/common"
+	"github.com/cloudflare/cfssl_trust/model/certdb"
+	"github.com/cloudflare/cfssl_trust/release"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var releaseCmd = &cobra.Command{
+	Use:   "release",
+	Short: "Display a list of the certificates in a release.",
+	Long:  "Display a list of the certificates in a release.",
+	Run:   listRelease,
+}
+
+func init() {
+	RootCmd.AddCommand(releaseCmd)
+}
+
+func listRelease(cmd *cobra.Command, args []string) {
+	switch len(args) {
+	case 0: // Don't do anything.
+	case 1:
+		_, err := release.Parse(args[0])
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "[!] Invalid release '%s'.\n", bundleRelease)
+			fmt.Fprintf(os.Stderr, "\tReason: %s\n", err)
+			os.Exit(1)
+		}
+		bundleRelease = args[0]
+	default:
+		fmt.Fprintln(os.Stderr, "[!] Too many arguments passed to 'release'.")
+	}
+
+	dbPath := viper.GetString("database.path")
+	db, err := sql.Open("sqlite3", dbPath)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "[!] %s\n", err)
+		os.Exit(1)
+	}
+
+	var rel *certdb.Release
+	if bundleRelease == "" {
+		rel, err = certdb.LatestRelease(db, bundle)
+	} else {
+		rel, err = certdb.FetchRelease(db, bundle, bundleRelease)
+	}
+
+	if err == sql.ErrNoRows {
```

Comment:
Not a blocker, but a preference - I usually like having the sql.ErrNoRows check within the db wrapper package (in this case, certdb). What do you think?